### PR TITLE
PLF-7962: CheckAll must be checked if all sub-checkboxes are checked

### PR DESCRIPTION
**Pull request by Jihed525**

### CODE REVIEW

File: integ-search-portlet/src/main/webapp/js/common/search.js [hunk 1]
```diff
@@ -621,6 +629,14 @@ window.initSearch = function initSearch() {
       window.search = search(); //perform search again to update the results
     });
 
+    //Check if all site Checkboxes are checked
+    $(document).on("click",":checkbox[name='site'][value!='all']", function(){
```
Comment (boubaker) [inline integ-search-portlet/src/main/webapp/js/common/search.js]: you should integrate this new algorithm to the algorithm of previous line (same on click event handling)
https://github.com/exodev/integration/blob/faf7b80552db1f391cb7096c7415b33df72a9610/integ-search-portlet/src/main/webapp/js/common/search.js#L618

File: integ-search-portlet/src/main/webapp/js/common/search.js [hunk 2]
```diff
@@ -607,6 +607,14 @@ window.initSearch = function initSearch() {
       window.search = search(); //perform search again to update the results
     });
 
+    //Check if all site Checkboxes are checked
+    $(document).on("click",":checkbox[name='contentType'][value!='all']", function(){
```
Comment (boubaker) [inline integ-search-portlet/src/main/webapp/js/common/search.js]: you should integrate this new algorithm to the algorithm of previous line (same on click event handling)
https://github.com/exodev/integration/blob/faf7b80552db1f391cb7096c7415b33df72a9610/integ-search-portlet/src/main/webapp/js/common/search.js#L596

File: integ-search-portlet/src/main/webapp/js/common/search.js (hunk 1 above)
Comment (Jihed525) [reply integ-search-portlet/src/main/webapp/js/common/search.js]: Done, Thank you!

File: integ-search-portlet/src/main/webapp/js/common/search.js (hunk 2 above)
Comment (Jihed525) [reply integ-search-portlet/src/main/webapp/js/common/search.js]: Done, Thank you!

File: integ-search-portlet/src/main/webapp/js/common/search.js (hunk 1 above)
Comment (Jihed525) [reply integ-search-portlet/src/main/webapp/js/common/search.js]: Done as discussed.

File: integ-search-portlet/src/main/webapp/js/common/search.js (hunk 2 above)
Comment (Jihed525) [reply integ-search-portlet/src/main/webapp/js/common/search.js]: Done as discussed.

**Comment by Jihed525** [comment]
> Could you restore the default file formatting ? 
Done.
> is it necessary to switch functions places ?
No need indeed to do so, i've restored the original functions place.

**Pull request merged**